docs/book/orders/orders.rst

```diff
@@ -85,7 +85,13 @@ In order to change the amount of items use the **OrderItemQuantityModifier**.
 
     $this->container->get('sylius.order_item_quantity_modifier')->modify($orderItem, 3);
 
-Add the item to the order. And then call the **CompositeOrderProcessor** on the order to have everything recalculated.
+Add the item to the order. And then call the **CompositeOrderProcessor** on the order to have
+everything recalculated. This **CompositeOrderProcessor** is a called multiple times in the checkout 
```

Comment:
Tl;dr: Let's put this graph after code example in the note box. Sample recommendation: 

```suggestion
everything recalculated. 

.. code-block:: php
    $order->addItem($orderItem);
    $this->container->get('sylius.order_processing.order_processor')->process($order);

.. note::

    This **CompositeOrderProcessor** is one of the most powerful concepts that handles order calculation heavy lifting. It is called multiple times in the checkout process, and internally it works like this:
    
    .. image: ../../_images/sylius_order_processor.png
        :align: center
        :scale: 70%
```